app/scripts/modules/core/src/managed/StatusBubble.less

```diff
@@ -1,39 +1,96 @@
 .StatusBubble {
-  display: grid;
-  grid-template-columns: 1fr;
-  grid-template-rows: 1fr;
-  border-radius: 50%;
+  display: flex;
+  flex-direction: row;
 
-  .status-bubble-icon-container {
+  .IconWrapper {
+    border-radius: 50%;
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-rows: 1fr;
+
+    &.withQuantity {
+      border-bottom-right-radius: 0%;
+    }
+  }
+
+  .IconContainer {
+    align-items: center;
+    display: flex;
+    flex-flow: row nowrap;
     grid-column: 1;
     grid-row: 1;
+    justify-content: center;
   }
 
-  &.status-bubble-inactive {
+  .status-inactive {
     background-color: var(--color-status-inactive);
   }
 
-  &.status-bubble-neutral {
+  .status-neutral {
     background-color: var(--color-status-neutral);
   }
 
-  &.status-bubble-info {
+  .status-info {
     background-color: var(--color-status-info);
   }
 
-  &.status-bubble-progress {
+  .status-progress {
     background-color: var(--color-status-progress);
   }
 
-  &.status-bubble-success {
+  .status-success {
     background-color: var(--color-status-success);
   }
 
-  &.status-bubble-warning {
+  .status-warning {
     background-color: var(--color-status-warning);
   }
 
-  &.status-bubble-error {
+  .status-error {
     background-color: var(--color-status-error);
   }
+
+  .Pill {
+    align-self: flex-end;
+    color: var(--color-icon-light);
+    display: inline-block;
+    font-family: 'Source Sans Pro';
+    font-weight: bold;
+    text-rendering: optimizeLegibility;
+    z-index: -1;
```

Comment:
The `pill`/`quantity` part of `StatusBubble` comes after the icon and I've used a negative margin and a lower `z-index` to make this pill move back and *under* the icon. Otherwise, the pill would cover some parts of the lower right corner of the icon.

Reply:
ah. yes ok thank you. all makes sense now.